codegen/sdk-codegen/build.gradle.kts

```diff
@@ -187,6 +165,8 @@ fun discoverServices(): List<AwsService> {
 }
 val discoveredServices: List<AwsService> by lazy { discoverServices() }
 
+val packageVersion = rootProject.file("Package.version.next").readText(Charset.forName("UTF-8")).trim()
+
```

Comment:
`packageVersion` is read from the `Package.version.next` file.  When codegen is performed, this file contains the SDK version being generated.

`packageVersion` is added to codegen settings for reference during codegen.